src/PresentationalComponents/StatusReports/AdvisoriesStatusReport.js

```diff
@@ -43,15 +43,21 @@ const StatusCard = ({ advisory: { attributes, id } }) =>
                                     {attributes.severity && (<GridItem>
                                         <Split hasGutter>
                                             <GridItem >
-                                                <SecurityIcon size="sm" color={advisorySeverities[attributes.severity].color} />
+                                                <Icon>
+                                                    <SecurityIcon color={advisorySeverities[attributes.severity].color} />
+                                                </Icon>
                                             </GridItem>
                                             <GridItem isFilled>{advisorySeverities[attributes.severity].label}</GridItem>
                                         </Split>
                                     </GridItem>)}
                                     {attributes.reboot_required && (
                                         <GridItem>
                                             <Split hasGutter>
-                                                <SplitItem><PowerOffIcon color='var(--pf-global--palette--red-100)' /></SplitItem>
+                                                <SplitItem>
+                                                    <Icon color='var(--pf-v5-global--palette--red-100)'>
```

Comment:
Because of this, the icon color seems to be affected
![image](https://github.com/RedHatInsights/patchman-ui/assets/59481011/2c207c5e-a109-40f6-822b-159c8489abd8)

Reply:
I think I will just replace it with `status: 'danger'` prop